helix-term/src/application.rs

```diff
@@ -866,9 +866,16 @@ impl Application {
         }));
 
         self.event_loop(input_stream).await;
-        self.close().await?;
+
+        let err = self.close().await.err();
```

Comment:
Small nitpick: shouldn't this be named `result` or `res`, not `err`?

Reply:
It's not a result; see [`Result::err`](https://doc.rust-lang.org/std/result/enum.Result.html#method.err).

Reply:
Ah, right.